src/amo/components/UserProfile/styles.scss

```diff
@@ -20,10 +20,15 @@ $avatar-size: 64px;
 }
 
 .UserProfile-avatar {
-  display: block;
```

Comment:
wrapped `img` in a `div`, so no need of `display:block`.
The wrapper will have the height, width and nothing goes out of wrapper div, image has `width:100%` and `height:auto` this way the image will always preserve to its original dimension ratio and also respect teh wrapper size.